tests/integration-tests/tests/iam/test_iam.py

```diff
@@ -363,6 +369,247 @@ def test_iam_resource_prefix(
             _test_iam_resource_in_cluster(cfn_client, iam_client, cluster.name, iam_resource_prefix)
 
 
+def _remove_from_config(config, section_list):
```

Comment:
It seems like you are doing the replacement manually.  Why not use a tool like jinja for this?